airflow/www_rbac/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "airflow",
+  "version": "0.9.0",
```

Comment:
??

2.0.0 ?

Reply:
Can we make this dynamic (ie. get it from version.py?)

Reply:
It doesn't look like this can be dyanmic. We can set it with `npm version X` so it could be scripted. But given we aren't going to publish this to npm's repo do we need a version at all? If we do can we set it to "will-never-be-published" or similar?

Reply:
From the [npm package.json docs](https://docs.npmjs.com/files/package.json):

> If you plan to publish your package, the most important things in your package.json are the name and version fields as they will be required.

That sounds like name and version could be left out in this case if we want.

Reply:
`name` and `version` is totally optional if we don't want to publish this as an npm package anytime in the future. 
The version number can not be dynamic or set automatically, as this is JSON file. One need to update this manually. 
New npm packages can not have uppercase letters or spaces in the name. (it could be `apache-airflow` though)

Reply:
It seems unlikely that we'd want to publish this -- its just the asset build pipeline, and I can't see a use case for someone else wanting to install this or depend upon it, so let's remove them.

Reply:
👍 that's true. will remove name and version from package.json in my next commit.

Reply:
Fixed. Removed name and version number from package.json